datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/VersionController.java

```diff
@@ -55,17 +67,25 @@
      */
     @SneakyThrows
     @GetMapping(value = "/versioned/{path:.*}", produces = APPLICATION_OCTET_STREAM_VALUE)
+    @ApiOperation("Read latest document from user's private space")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Document was successfully read"),
+            @ApiResponse(code = 403, message = "Access denied"),
```

Comment:
403 has default message, not necessary to add it

Reply:
removed